### DISCUSSION TITLE
Ripple effect on line layer

### DISCUSSION THREAD

**Discussion by Nitikaa** [visgl/deck.gl#7295] [Q&A]
Is there any way to add ripple effect for selected line in LineLayer? I'm trying to highlight selected object with some animation effects. like [https://docs.mapbox.com/mapbox-gl-js/example/add-image-animated](https://github.com/visgl/deck.gl/issues/url)

**Comment by AdriSolid** [comment]
[This codepen](https://codepen.io/AdriSolid/pen/bGMYJWB) can be useful. Based on [this one](https://codepen.io/Pessimistress/pen/jOqVKVB) by @Pessimistress

**Reply by Nitikaa** [reply]
transitions property isn't working for me. It just shows line like this 
![image](https://user-images.githubusercontent.com/7350906/193009916-1bc34eb7-0f26-4f0f-8b9e-e8af7dc3e8d3.png)

**Reply by AdriSolid** [reply]
Try to reproduce your issue in a codepen please

**Reply by Nitikaa** [reply]
With this approach the layers are rerenfering with large data set but I wanted just one selected layer to be animated.

**Reply by Nitikaa** [reply]
Re rendering *

**Reply by AdriSolid** [reply]
Probably not the best solution as this is cloning and filtering two layer instances, one for the selected layer with the effect and other for the rest, but could work: https://codepen.io/AdriSolid/pen/wvjZPRx
